Apsitvarkom.DataAccess/PollutedLocationDTOFileRepository.cs

```diff
@@ -77,7 +77,7 @@ public static PollutedLocationDTOFileRepository FromContent(IMapper mapper, stri
     /// <returns>Path on success, <see cref="FormatException"/> otherwise.</returns>
     private static string CheckIfFileIsJson(string path)
     {   
-        const string regexPattern = @"(\.json)$";
+        const string regexPattern = @"(^[a-zA-Z0-9\.json-]+)$";
```

Comment:
https://regex101.com/ Try experimenting in this page with your regex pattern.
It seems wrong:
- filename.jsona - is a match
- folder/filename.json - is not a match
- folder\filename.json - is not a match
- C:/filename.json - is not a match